Library/Homebrew/cmd/uninstall.rb

```diff
@@ -6,44 +6,58 @@
 
 require "keg"
 require "formula"
+require "diagnostic"
 require "migrator"
 
 module Homebrew
   def uninstall
     raise KegUnspecifiedError if ARGV.named.empty?
 
-    if !ARGV.force?
-      ARGV.kegs.each do |keg|
-        keg.lock do
-          puts "Uninstalling #{keg}... (#{keg.abv})"
-          keg.unlink
-          keg.uninstall
-          rack = keg.rack
-          rm_pin rack
-
-          if rack.directory?
-            versions = rack.subdirs.map(&:basename)
-            verb = versions.length == 1 ? "is" : "are"
-            puts "#{keg.name} #{versions.join(", ")} #{verb} still installed."
-            puts "Remove all versions with `brew uninstall --force #{keg.name}`."
-          end
-        end
-      end
-    else
-      ARGV.named.each do |name|
+    kegs_by_rack = if ARGV.force?
+      Hash[ARGV.named.map { |name|
```

Comment:
AFK so can't verify right now, but I believe the new RuboCop rules enforce the semantic style for blocks, rather than always using `do...end` for multiline.